src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.csproj

```diff
@@ -6,10 +6,12 @@
     <ShouldWriteSigningRequired>false</ShouldWriteSigningRequired>
     <AllowReferenceFromRuntime>true</AllowReferenceFromRuntime>
     <RuntimeProjectFile>$(ProjectDir)\external\test-runtime\XUnit.Runtime.depproj</RuntimeProjectFile>
+    <PackageTargetFramework>netstandard2.0;$(UAPvNextTFM)</PackageTargetFramework> 
```

Comment:
@ericstj looks like we still need to do things like this since compatibility between UAP and netstandard2.0 is not yet inferred. Is updating the nuget mappings enough to fix this?

Reply:
Update the NuGet dlls used by build-tools: specifically NuGet.Frameworks.dll.  That's where the mapping lives.